src/Client/app/(root)/home/page.tsx

```
@@ -1,52 +1,68 @@
+  const requestPermissions = async () => {
+    try {
+      if (document.visibilityState !== 'visible') {
+        alert('Please keep the tab active and focused when granting permissions.');
+        return;
+      }
+      await navigator.mediaDevices.getUserMedia({ video: true, audio: true });
+      await navigator.mediaDevices.getDisplayMedia({ video: true });
+      setPermit(true);
+    } catch (err) {
+      console.error('Permission denied or failed:', err);
+      alert('Permission denied or failed: ' + (err as Error).message);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)